src/bulma/parts/FormContent.vue

```diff
@@ -1,7 +1,7 @@
 <template>
     <div>
         <form-header/>
-        <form class="is-marginless">
+        <form class="is-marginless" @submit.prevent="">
```

Comment:
do we need the `""`

Reply:
I thought we did, but looks like we don't after all.